From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 12 Jan 2020 17:32:13 +0000
Subject: ALSA: compress: Avoid ABI change in 3.16.81
Forwarded: not-needed

Including <linux/compat.h> adds type definitions that weren't visible
to genksyms before.  There hasn't been any real ABI change, so hide
the #include from genksyms.

---
--- a/sound/core/compress_offload.c
+++ b/sound/core/compress_offload.c
@@ -38,7 +38,9 @@
 #include <linux/uio.h>
 #include <linux/uaccess.h>
 #include <linux/module.h>
+#ifndef __GENKSYMS__
 #include <linux/compat.h>
+#endif
 #include <sound/core.h>
 #include <sound/initval.h>
 #include <sound/compress_params.h>
